src/atoms/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,55 @@
+import { padding } from 'polished';
+import React, { Component, createRef, ReactNode } from 'react';
+
+import styled from '_styled-components';
+import { borderRadius, scale } from 'Theme';
+
+const Relative = styled.div`
+  position: relative;
+`;
+
+const InnerTooltip = styled.div<{ height?: number }>`
+  border: 0.0625em solid #bac9d4;
+  background: ${props => props.theme.controlBackground};
+  border-radius: ${borderRadius};
+  box-shadow: 0 0.1875em 0.375em 0 #00000012,
+    0 0.4375em 0.625em 0.4375em #32325d1a;
+  ${padding(scale(-1))};
+  position: absolute;
+  bottom: ${props => props.height}px;
```

Comment:
Should we use `em` here?

Reply:
Alright! I was wondering about if that would be an issue so that makes sense. Thanks Nick!